cloudformationTemplates/urlPreviewForAsyncChat/cloudformation.yaml

```diff
@@ -104,6 +125,7 @@ Resources:
   ### DDB for url preview cache ###
   UrlPreviewCacheTable:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The deployment risk in not dynamically adjusting the provisioned throughput capacity of DynamoDB tables is the potential for increased costs and performance issues. If the application&#39;s traffic patterns change, and the DynamoDB tables are not scaled accordingly, the organization may end up over-provisioning or under-provisioning the throughput capacity. This can lead to higher than necessary costs or, conversely, performance degradation and service disruptions. Failing to address this risk can result in wasted resources and operational challenges during the deployment and ongoing maintenance of the application.